app/controllers/settings_controller.rb

```diff
@@ -21,9 +21,10 @@ def update
       SiteSettings.model_tags_stop_words_locale = params[:model_tags][:stop_words_locale]
       SiteSettings.model_tags_custom_stop_words = params[:model_tags][:custom_stop_words].split
       SiteSettings.model_tags_auto_tag_new = params[:model_tags][:auto_tag_new]
-      SiteSettings.model_path_prefix_template = params[:model_tags][:model_path_prefix_template]
-      SiteSettings.model_tags_tag_model_path_prefix = params[:model_tags][:model_tags_tag_model_path_prefix]
-      SiteSettings.model_path_suffix_model_id = params[:model_tags][:model_path_suffix_model_id]
+    end
+    if current_user.admin? && params[:folders]
+      SiteSettings.model_path_template = params[:folders][:model_path_template]
```

Comment:
I suggest we force this to have a leading / or not.   Right now it could be either way, which at the very least ends with odd-looking results in the dialog box for organizing files.

Reply:
Upon further inspection... if you have a leading slash, it will always think your files are not organized... so strip any leading slash here.

Reply:
done!